app/controllers/cart_items_controller.rb

```diff
@@ -0,0 +1,84 @@
+class CartItemsController < ApplicationController
+  skip_before_action :ensure_user_is_owner
+  skip_before_action :ensure_user_is_owner_or_clerk
+
+  def create
+    user_cart_id = Cart.get_current_user_cart(@current_user.id).id
+    menu_item_info = params[:menu_item_info]
+    cart_total = Cart.get_current_user_cart(@current_user.id).total
+
+    menu_item_info.each do |item|
+      id = item["info"]["menu_item_id"] ? item["info"]["menu_item_id"] : nil
+      if id
+        cart_item = CartItem.new(
+          cart_id: user_cart_id,
+          menu_item_id: id,
+          menu_item_name: MenuItem.find(id).name,
+          menu_item_price: MenuItem.find(id).price,
+          menu_item_quantity: item["info"]["quantity"],
```

Comment:
**Suggestion:** Avoid querying the database multiple times for the same data. You can store the result of `MenuItem.find(id)` in a variable and reuse it.
```suggestion
        menu_item = MenuItem.find(id)
        cart_item = CartItem.new(
          cart_id: user_cart_id,
          menu_item_id: id,
          menu_item_name: menu_item.name,
          menu_item_price: menu_item.price,
          menu_item_quantity: item["info"]["quantity"],
        )
```